src/components/side-panel/side-panel.styles.tsx

```diff
@@ -0,0 +1,49 @@
+import { Color } from "@lifesg/react-design-system/color";
+import { Transition } from "@lifesg/react-design-system/transition";
+import styled, { css } from "styled-components";
+
+// =============================================================================
+// STYLING INTERFACE
+// =============================================================================
+export interface IWrapperStyleProps {
+    $isCollapsed: boolean;
+}
+
+// =============================================================================
+// STYLING
+// =============================================================================
+export const Wrapper = styled.div<IWrapperStyleProps>`
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 36.8rem;
+    transition: ${Transition.Base};
+    background: ${Color.Neutral[8]};
+    border-left: 1px solid ${Color.Neutral[5]};
+    margin-left: auto;
+    box-shadow: 0px 2px 12px 0px rgba(104, 104, 104, 0.25);
+    position: fixed;
```

Comment:
as the header and title bar are separated from this library, we are not able to accommodate their heights here. with that, we should switch from fixed positioning to sticky positioning to keep the panel at the same position when there is scrolling.

Reply:
my apologies, it appears that the sticky positioning is not working well, can you revert to fixed positioning first? we will come up with an alternate solution to this in another ticket.